packages/react-examples/src/react/ChoiceGroup/ChoiceGroup.Image.Example.tsx

```diff
@@ -6,17 +6,18 @@ const options: IChoiceGroupOption[] = [
   {
     key: 'bar',
     imageSrc: TestImages.choiceGroupBarUnselected,
-    imageAlt: 'Bar chart icon',
+    imageAlt: 'Bar chart',
     selectedImageSrc: TestImages.choiceGroupBarSelected,
     imageSize: { width: 32, height: 32 },
-    text: 'Clustered bar chart', // This text is long to show text wrapping.
+    text: 'Bar', // This text is long to show text wrapping.
```

Comment:
Should the comment after be removed now?

Reply:
ah, yup, fixed! 😄